extensions/thunder/strategies/thunder_fsdp.py

```diff
@@ -166,7 +166,7 @@ def setup_module(self, module: Module) -> Module:
                 )
             assert cd.is_module  # sanity check
             fsdp_module = thunder.distributed.fsdp(
-                cd.fn,
+                cd._thunder_module_map[id(cd.fn)],
```

Comment:
I think we want to drop calling fsdp in favour of calling thunder.jit in the Strategy.
I'll chat you up on this.